mobile/android/base/android_strings.dtd

```diff
@@ -508,28 +475,23 @@ it is applicable. -->
                                          z systemu Android">
 <!ENTITY bookmarkhistory_import_wait "Proszę czekać…">
 
-<!-- Localization note (suggestion_for_engine): The placeholder &formatS1; will be
-     replaced with the name of the search engine. The placeholder &formatS2; will be
-     replaced with the search query. -->
 <!ENTITY suggestion_for_engine "Szukaj „&formatS2;” za pomocą &formatS1;">
 
 <!ENTITY webapp_generic_name "Aplikacja">
 
 <!ENTITY searchable_description "Zakładki i historia">
 
-<!-- Updater notifications -->
 <!ENTITY updater_start_title2 "&brandShortName;: dostępne są nowe aktualizacje">
 <!ENTITY updater_start_select2 "Dotknij, aby zaktualizować">
 
 <!ENTITY updater_downloading_title2 "Pobieranie: &brandShortName;">
-<!ENTITY updater_downloading_title_failed2 "Pobieranie nie udało się">
+<!ENTITY updater_downloading_title_failed2 "Pobieranie się nie udało">
```

Comment:
For me [it is not that obvious](https://github.com/piotrdrag/mozilla-l10n-pl/commit/af9cbfe988226b34a860ed0bd937eecc50b7f103), see Aviary.pl bugs [4769](http://bugs.aviary.pl/show_bug.cgi?id=4769), [2645](http://bugs.aviary.pl/show_bug.cgi?id=2645), [2811](http://bugs.aviary.pl/show_bug.cgi?id=2811) and [1074](http://bugs.aviary.pl/show_bug.cgi?id=1074) among others.